test/runtests.jl

```diff
@@ -202,7 +202,7 @@ end
                 # type inference checks of `rrule` fail on old Julia versions
                 # for higher-dimensional arrays:
                 # https://github.com/JuliaMath/AbstractFFTs.jl/pull/58#issuecomment-916530016
-                check_inferred = ndims(x) < 3 || VERSION >= v"1.6"
+                check_inferred = ndims(x) < 3
```

Comment:
Is this the only occurrence of a now unnecessary `VERSION` check?

Reply:
I did a find for "VERSION" and this is the only place it came up.